packages/driver/cypress/e2e/e2e/origin/cookie_behavior.cy.ts

```diff
@@ -1285,6 +1285,7 @@ describe('Cookie Behavior', { browser: '!webkit' }, () => {
 
         describe('misc', () => {
           describe('domains', () => {
+            // NOTE: tested in ./with-inject-document-domain/cookie_behavior
```

Comment:
does this file exist? I'm having issues finding it in the branch.

Reply:
Not anymore!